libsolidity/interface/StandardCompiler.cpp

```diff
@@ -564,12 +564,12 @@ std::optional<Json> checkOutputSelection(Json const& _outputSelection)
 
 /// Validates the optimizer settings and returns them in a parsed object.
 /// On error returns the json-formatted error message.
-std::variant<OptimiserSettings, Json> parseOptimizerSettings(Json const& _jsonInput)
+std::variant<OptimiserSettings, Json> parseOptimizerSettings(std::string_view const _language, Json const& _jsonInput)
```

Comment:
Just curious: is `const` here redundant, since `string_view` cannot mutate the string?

Reply:
Ah, but I guess you could change the `string_view` itself, by moving its "window"...

Reply:
Yup exactly :)